diff --git a/node_modules/oao/lib/utils/readSpecs.js b/node_modules/oao/lib/utils/readSpecs.js
index 729b985..8a99d85 100644
--- a/node_modules/oao/lib/utils/readSpecs.js
+++ b/node_modules/oao/lib/utils/readSpecs.js
@@ -74,16 +74,16 @@ const validatePkgName = (pkgPath, name) => {
   }
 
   if (pkgPath === '.') return;
-  const segments = pkgPath.split('/');
-
-  if (name[0] !== '@' && name !== segments[segments.length - 1]) {
-    const errMsg = `Package name (${name}) does not match directory name ${pkgPath}`;
-
-    _storyboard.mainStory.error(errMsg);
-
-    const err = new Error('INVALID_DIR_NAME'); // $FlowFixMe (piggyback on exception)
-
-    err.details = errMsg;
-    throw err;
-  }
+  // const segments = pkgPath.split('/');
+
+  // if (name[0] !== '@' && name !== segments[segments.length - 1]) {
+  //   const errMsg = `Package name (${name}) does not match directory name ${pkgPath}`;
+  // 
+  //   _storyboard.mainStory.error(errMsg);
+  // 
+  //   const err = new Error('INVALID_DIR_NAME'); // $FlowFixMe (piggyback on exception)
+  // 
+  //   err.details = errMsg;
+  //   throw err;
+  // }
 };
\ No newline at end of file
